test/over_react_test/custom_matchers_test.dart

```diff
@@ -798,6 +825,16 @@ main() {
     test('logsPropError', () {
       expect(() => mount((Sample()..shouldNeverBeNull = false)()),
           logsPropError('shouldNeverBeNull', 'should not be false'));
+
+      if (!runtimeSupportsPropTypeWarnings()) {
+        // The expectation is purposefully faulty so that we can assert the in dart2js runtimes,
```

Comment:
#nit
```suggestion
        // The expectation is purposefully faulty so that we can assert that in dart2js runtimes,
```

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/5e736cd83aa9091f0e0dd699c9e3d8020ce7fd4f